src/backend/app/submission/submission_crud.py

```diff
@@ -436,6 +443,99 @@ def extract_files(zip_file_path):
     return final_zip_file_path
 
 
+def update_submission_in_s3(
+    db: Session, project_id: int, background_task_id: uuid.UUID
+):
+    try:
+        # Get Project
+        get_project_sync = async_to_sync(project_crud.get_project)
+        project = get_project_sync(db, project_id)
+
+        # Upload submission in s3
+        submission = get_all_submissions(db, project_id)
+        submission_path = f"/{project.organisation_id}/{project_id}/submission.zip"
+        file_obj = BytesIO(json.dumps(submission).encode())
+
+        # Metadata
+        # ODK Credentials
+        odk_credentials = project_schemas.ODKCentral(
+            odk_central_url=project.odk_central_url,
+            odk_central_user=project.odk_central_user,
+            odk_central_password=project.odk_central_password,
+        )
+        from sqlalchemy.sql import text
```

Comment:
Try not to put imports directly in functions.
They should always be at the top of the file, except some specific scenarios with optional dependencies.